scripts/cmake/FindIOKit.cmake

```diff
@@ -14,7 +14,7 @@
 #  IOKit_INCLUDE_DIRS - the IOKit include directories 
 #  IOKit_LIBRARIES - link these to use IOKit
 
-include(FindPkgMacros)
+include(${CMAKE_CURRENT_SOURCE_DIR}/scripts/cmake/FindPkgMacros.cmake)
```

Comment:
These are not scripts, so let's not put them into `scripts`.

A common approach is to create ` cmake` folder in the project root for global CMake modules.

Example:
https://github.com/stenzek/duckstation

Or when the modules are not global, just put them in subfolder of the parts of the codebase they apply to:

Example:
https://github.com/pabloariasal/modern-cmake-sample/tree/master/libjsonutils/cmake

So in this case, I think the best place for them would be `src/libs/manymouse/cmake`.

Reply:
OK, moved to `src/libs/manymouse/cmake`.